crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -781,9 +813,90 @@ impl LogsController {
             metadata_store_client,
             metadata_writer,
             async_operations: JoinSet::default(),
+            tail_updates_tx,
+            in_flight_refresh_tail: None,
```

Comment:
Maybe for consistency?
```suggestion
            in_flight_refresh_tails: None,
```